package.json

```diff
@@ -32,49 +32,50 @@
     "postpublish": "ember ts:clean"
   },
   "dependencies": {
-    "@ember-decorators/babel-transforms": "^3.1.0",
-    "ember-cli-babel": "^7.1.0",
-    "ember-decorators": "^3.1.4"
+    "@ember-decorators/babel-transforms": "^5.1.4",
+    "ember-cli-babel": "^7.6.0",
+    "ember-cli-typescript": "2.0.0",
```

Comment:
@charlesdemers Is there a reason this has been moved to dependencies instead of staying a dev dependency?

Reply:
Yes! The `ember-cli-typescript` docs say that addons should install it as a dependency instead of a dev dependency: https://typed-ember.github.io/ember-cli-typescript/versions/master/docs#installing-in-addons